vllm/v1/worker/gpu_model_runner.py

```diff
@@ -602,269 +633,3 @@ def _get_padded_batch_size(self, batch_size: int) -> Optional[int]:
             if batch_size <= size:
                 return size
         return None
-
```

Comment:
Refactor : Moved CachedRequestState and InputBatch to input_batch.py. It looked like a good refactor to reduce file-size. In this PR it lets both `gpu_model_runner.py` and `lora_gpu_model_runner_mixin.py` import these datastructures from InputBatch.